.pre-commit-config.yaml

```diff
@@ -74,10 +74,6 @@ repos:
     - id: check-yaml
     - id: end-of-file-fixer
     - id: trailing-whitespace
-
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.9.2
-    hooks:
```

Comment:
I would favor adding this back in and changing the link to `github.com`, since the version tagged in `pre-commit-hooks` might differ from `v2.3.0`.

Reply:
Interesting, I didn't know there was a github.com version.

Reply:
I suspect they might have moved it over to github recently.